yt_dlp/extractor/zaiko.py

```diff
@@ -90,3 +92,27 @@ def _real_extract(self, url):
                 'thumbnail': ('poster_url', {url_or_none}),
             })),
         }
+
+
+class ZaikoETicketIE(ZaikoBaseIE):
+    _VALID_URL = r'https?://zaiko\.io/account/eticket/(?P<id>\w{48}=)'
+    _TESTS = [{
+        'url': 'https://zaiko.io/account/eticket/000000000000000000000000000000000000000000000000=',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        ticket_id = self._match_id(url)
+
+        webpage, urlh = self._download_webpage_handle(url, ticket_id)
+        final_url = urlh.geturl()
+        if 'zaiko.io/login' in final_url:
+            self.raise_login_required()
+        eticket_meta = self._parse_vue_element_attr('eticket', webpage, ticket_id)
+
+        ticket_details = eticket_meta.get('ticket-details')
+        streams = eticket_meta.get('streams') or []
+
+        return self.playlist_result(
+            [self.url_result(stream.get('url'), ZaikoIE) for stream in streams], ticket_id,
+            ticket_details.get('event_name'), thumbnail=ticket_details.get('event_img_url'))
```

Comment:
`traverse_obj` will prevent `url_result(None)` and can eliminate the need for the intermediate vars
```suggestion
        eticket = self._parse_vue_element_attr('eticket', webpage, ticket_id)

        return self.playlist_result(
            [self.url_result(stream, ZaikoIE) for stream in traverse_obj(eticket, ('streams', ..., 'url'))],
            ticket_id, **traverse_obj(eticket, ('ticket-details', {
                'title': 'event_name',
                'thumbnail': 'event_img_url',
            })))
```

Reply:
Wow! Again, a magic "```traverse_obj```". Thank you!